README.md

```diff
@@ -34,4 +34,21 @@ it('uses any data passed as a second argument to fill out the form', () => {
       assert.equal(name, 'David HasselHoff');
     });
 });
+
+it('saves screenshots of errors to specified screenshot location', () => {
+  const inputs = {};
+  return browser.goto('/confirm', inputs, { screenshots: '/path/to/output/dir' });
+});
+
+it('tries a pre-specified number of times to get past stuck loops', () => {
+  const inputs = {};
+  return browser.goto('/confirm', inputs, { maxLoops: 1 });
+});
 ```
+
+## Options
+
+Options are passed as a third argument to the exposed method. The following options are available:
+
+* `maxLoops` - determines how many times a step will retry if it resolves back to itself on submission before failing. Default: `3`
+* `screenshots` - specifies a location to save screenshots of the page when it gets stuck. If not specified then no screenshots are saved.
```

Comment:
That's mostly just debug code because otherwise I was flying totally blind in phantom, but I figured it's probably long-term useful.